queries.sql

```diff
@@ -7,4 +7,11 @@ SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name != 'Gabumon';
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.3 AND 17.4;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.3 AND 17.4;
+
```

Comment:
- [ ] Kindly make sure you create queries to answer the following questions in transactions as required for the project:
  - Inside a transaction update the `animals` table by setting the `species` column to `unspecified`. Verify that change was made. Then roll back the change and verify that the `species` columns went back to the state before the transaction.
  - Inside a transaction:
  - Update the `animals` table by setting the `species` column to `digimon` for all animals that have a name ending in `mon`.
  - Update the `animals` table by setting the `species` column to `pokemon` for all animals that don't have `species` already set.
  - Commit the transaction.
  - Verify that change was made and persists after commit.

- [ ] Once you made the changes above, make sure you add screenshots of the transition results to the pull request description